src/getting_started/changelogs/Changelogs.js

```diff
@@ -14,6 +14,100 @@ export default function Authentication() {
         API V4 since 2017-09-18.  Please check here regularly for updates.
         </p>
       </section>
+      <section>
+        <h2>2018-01-08</h2>
+        <hr /><br />
+        <b>Breaking:</b><br />
+        <ul>
+          <li>Update /linode/distributions and /image responses
+            <ul>
+              <li>remove status key</li>
+              <li>remove filesystem key</li>
+              <li>remove last_used key</li>
+              <li>rename creator -> created_by (string)</li>
+              <li>rename min_deploy_size -> size (int)</li>
+            </ul>
+          </li>
+        </ul>
+        <b>Features:</b><br />
+        <ul>
+          <li>Added enable, disable managed service endpoints
+            <ul>
+              <li>Added POST /managed/services/:id/enable</li>
+              <li>Added POST /managed/services/:id/disable</li>
+              <li>Added status to managed service JSON response object</li>
+            </ul>
+          </li>
+          <li>Added GET /managed/stats endpoint</li>
+          <li>Added PUT for /managed/linode_settings</li>
+          <li>Added PUT endpoints to managed service</li>
+          <li>Add managed_issue_type to support/tickets/
+            <ul>
+              <li>GET /support/tickets returns new managed_issue_type key</li>
+              <li>POST /support/tickets now accepts this key if the account has
+              Managed Services enabled</li>
+            </ul>
+          </li>
+          <li>Added GET /managed/issues and GET /managed/issues/:id
+            <ul>
+              <li>GET /managed/issues returns all issues, including relevant history</li>
+              <li>GET /managed/issues/:id returns a single issue</li>
+              <li>Since managed issues are backed by support tickets, uses the ticket ID</li>
+            </ul>
+          </li>
+          <li> Added vat_number to  /account/settings
+            <ul>
+              <li>Can be viewed and updated</li>
+            </ul>
+          </li>
+          <li>Sends emails when enabling/disabling TFA</li>
+          <li>Rendered zone file returned in Domain object</li>
+          <li>Added DELETE /managed/services/:id endpoint</li>
+          <li>Added DELETE /managed/contacts/:id</li>
+        </ul>
+        <b>Bugfixes:</b><br />
+        <ul>
+          <li>Allow filtering of GET /images</li>
+        </ul>
+        <h2>2017-11-20</h2>
+        <hr /><br />
+        <b>Breaking Changes:</b><br />
+        <ul>
+          <li>Reworked UserGrant system
+            <ul>
+              <li>Three Grant levels are now enforced: No grants, read_only, and read_write</li>
```

Comment:
Should we put quotes around read_only and read_write? We do so below.

Reply:
These aren't new ?